### PR TITLE
NMS-13319: Add OpenAPI doc for health-check

### PR DESCRIPTION
**Pull request by jadyhou**
Add OpenAPI doc for health-check

### External References

* JIRA (Issue Tracker): https://issues.opennms.org/browse/NMS-13319

### CODE REVIEW

**Comment by opennms-bamboo** [comment]
Welcome Jane Hou (jhou@opennms.com).

We would like to thank you for your contribution.

I could not find a signed OpenNMS Contributor Agreement (OCA) assigned to the username Jane Hou (jhou@opennms.com).
In order to maintain some order with respect to the copyright to the code,
every contributor is asked to sign a "Contributor Agreement".

The OpenNMS Contributor Agreement can be downloaded [here](https://wiki.opennms.org/wiki/images/6/6b/ContributorAgreement.pdf).
Please send a signed copy to oca@opennms.org.

If you need more information, please visit https://wiki.opennms.org/wiki/Contributor_Agreement

**Comment by opennms-bamboo** [comment]
Welcome Jane Hou (jhou@opennms.com).

We would like to thank you for your contribution.

I could not find a signed OpenNMS Contributor Agreement (OCA) assigned to the username Jane Hou (jhou@opennms.com).
In order to maintain some order with respect to the copyright to the code,
every contributor is asked to sign a "Contributor Agreement".

The OpenNMS Contributor Agreement can be downloaded [here](https://wiki.opennms.org/wiki/images/6/6b/ContributorAgreement.pdf).
Please send a signed copy to oca@opennms.org.

If you need more information, please visit https://wiki.opennms.org/wiki/Contributor_Agreement

**Comment by opennms-bamboo** [comment]
Welcome Jane Hou (jhou@opennms.com).

We would like to thank you for your contribution.

I could not find a signed OpenNMS Contributor Agreement (OCA) assigned to the username Jane Hou (jhou@opennms.com).
In order to maintain some order with respect to the copyright to the code,
every contributor is asked to sign a "Contributor Agreement".

The OpenNMS Contributor Agreement can be downloaded [here](https://wiki.opennms.org/wiki/images/6/6b/ContributorAgreement.pdf).
Please send a signed copy to oca@opennms.org.

If you need more information, please visit https://wiki.opennms.org/wiki/Contributor_Agreement

File: core/health/rest/src/main/resources/openapi/health-check-api.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+info:
+  version: 1.0.0
+  title: OpenNMS Health Check RESTful API
+  description: OpenNMS Health Check RESTful API
+  contact:
+    name: OpenNMS
+    url: 'http://www.opennms.com/'
+  license:
+    name: OpenNMS(R) Licensing
+    url: 'http://www.gnu.org/licenses/'
```
Comment (RangerRick) [inline core/health/rest/src/main/resources/openapi/health-check-api.yaml:11]: This is pretty generic, it should probably point to https://github.com/OpenNMS/opennms/blob/develop/LICENSE.md instead

**Comment by opennms-bamboo** [comment]
Welcome @jadyhou.

We would like to thank you for your contribution.

I could not find a signed OpenNMS Contributor Agreement (OCA) assigned to the username @jadyhou.
In order to maintain some order with respect to the copyright to the code,
every contributor is asked to sign a "Contributor Agreement".

The OpenNMS Contributor Agreement can be downloaded [here](https://wiki.opennms.org/wiki/images/6/6b/ContributorAgreement.pdf).
Please send a signed copy to oca@opennms.org.

If you need more information, please visit https://wiki.opennms.org/wiki/Contributor_Agreement

**Comment by opennms-bamboo** [comment]
Welcome Jane Hou (jhou@opennms.com).

We would like to thank you for your contribution.

I could not find a signed OpenNMS Contributor Agreement (OCA) assigned to the username Jane Hou (jhou@opennms.com).
In order to maintain some order with respect to the copyright to the code,
every contributor is asked to sign a "Contributor Agreement".

The OpenNMS Contributor Agreement can be downloaded [here](https://wiki.opennms.org/wiki/images/6/6b/ContributorAgreement.pdf).
Please send a signed copy to oca@opennms.org.

If you need more information, please visit https://wiki.opennms.org/wiki/Contributor_Agreement

File: core/health/rest/src/main/resources/openapi/health-check-api.yaml (hunk 1 above)
Comment (jadyhou) [reply core/health/rest/src/main/resources/openapi/health-check-api.yaml]: Thanks, updated, please check. I also changed the branch that is going to merge from develop to release-28.x

**Review by RangerRick — APPROVED** [review]

**Comment by opennms-bamboo** [comment]
Welcome Jane Hou (jhou@opennms.com).

We would like to thank you for your contribution.

I could not find a signed OpenNMS Contributor Agreement (OCA) assigned to the username Jane Hou (jhou@opennms.com).
In order to maintain some order with respect to the copyright to the code,
every contributor is asked to sign a "Contributor Agreement".

The OpenNMS Contributor Agreement can be downloaded [here](https://wiki.opennms.org/wiki/images/6/6b/ContributorAgreement.pdf).
Please send a signed copy to oca@opennms.org.

If you need more information, please visit https://wiki.opennms.org/wiki/Contributor_Agreement

**Pull request merged**